pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -584,7 +584,7 @@ cdef inline int64_t parse_iso_format_string(str ts) except? -1:
         bint have_dot = 0, have_value = 0, neg = 0
         list number = [], unit = []
 
-    err_msg = "Invalid ISO 8601 Duration format - {}".format(ts)
+    err_msg = "Invalid ISO 8601 Duration format - {ts}"
```

Comment:
Missing the f

Reply:
How dumb I am !  I will propose a new (valid) PR tomorow.
Thank for the reply